lazylint

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
You should create a script to ignore ;-)

Reply:
I really should...